e2e/next/src/next.test.ts

```diff
@@ -342,7 +342,12 @@ describe('Next.js Applications', () => {
     updateFile(
       `apps/${appName}/next.config.js`,
       `
-      module.exports = {}
+      module.exports = {
+        future: {
+          // Nx doesn't support webpack 5 yet
+          webpack5: false,
+        }
+      }
```

Comment:
this change is required, as next.js now enables webpack 5 if you don't have any custom webpack configuration